flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why no aarch64?

Reply:
because i'm unable to generate with Godot the aarch64 in godot 2.1.
My plan is to support it later